src/sections/dataset/publish-dataset/PublishDatasetModal.tsx

```diff
@@ -23,6 +23,8 @@ interface PublishDatasetModalProps {
   repository: DatasetRepository
   persistentId: string
   releasedVersionExists: boolean
+  nextMajorVersion: string | undefined
+  nextMinorVersion: string | undefined
   handleClose: () => void
```

Comment:
`nextMajorVersion: string | undefined` could be changed to `nextMajorVersion?: string`  ?
I believe we are using that convention in the rest of the project?

Same with `nextMinorVersion` prop